downstream/modules/navigator/con-navigator-modes.adoc

```diff
@@ -34,3 +34,5 @@ colon commands:: You can access all the {Navigator} commands with a colon, such
 navigating the text-based interface:: The screen shows how to page up or down, scroll, escape to a prior screen or access `:help`.
 output by line number:: You can access any line number in the displayed output by preceding it with a colon, for example `:12`.
 color-coded output:: With colors enabled, {Navigator} displays items such as deprecated modules in red.
+
```

Comment:
There's a typo in the lead-in sentence for the table in ref-navigator-config-settings.adoc:
"The following table describe each parameter"